x-pack/plugins/observability/public/components/custom_threshold/components/alert_flyout.tsx

```diff
@@ -34,23 +28,15 @@ export function AlertFlyout(props: Props) {
         onClose: onCloseFlyout,
         canChangeTrigger: false,
         ruleTypeId: OBSERVABILITY_THRESHOLD_RULE_TYPE_ID,
-        metadata: {
-          currentOptions: props.options,
-          series: props.series,
-        },
         validConsumers: observabilityRuleCreationValidConsumers,
         useRuleProducer: true,
       }),
-    // eslint-disable-next-line react-hooks/exhaustive-deps
     [triggersActionsUI, onCloseFlyout]
   );
 
   return <>{visible && AddAlertFlyout}</>;
 }
 
 export function PrefilledThresholdAlertFlyout({ onClose }: { onClose(): void }) {
-  const { metricThresholdPrefill } = useAlertPrefillContext();
-  const { groupBy, filterQuery, metrics } = metricThresholdPrefill;
-
-  return <AlertFlyout options={{ groupBy, filterQuery, metrics }} visible setVisible={onClose} />;
+  return <AlertFlyout visible setVisible={onClose} />;
```

Comment:
We don't need to send `options={{ groupBy, filterQuery, metrics }}` ?

Reply:
This is related to preFill logic; I removed it because it did not match the new custom threshold structure and created a ticket to add this logic correctly (https://github.com/elastic/kibana/issues/170295)